infrequent-env

```diff
@@ -2,6 +2,8 @@
 
 export PATH=`pwd`/scripts:${PATH}
 
+# export CPU_ARCH=aarch64
```

Comment:
do we want to keep this here? I believe this is already configured at setup-env.template?

Reply:
Great point,  I'll yank it completely though,  I think it may be something I made up until I found PLATFORM.